shielddash/urls.py

```diff
@@ -17,10 +17,11 @@
 from django.contrib import admin
 from django.views.generic import TemplateView
 
-from .studies.views import StudyListView
+from .studies.views import StudyListView, study_detail
 
 
 urlpatterns = [
+    url(r'^studies/(?P<study_id>\d+)/', study_detail, name='study-detail'),
     url(r'^studies/', StudyListView.as_view(), name='study-list'),
```

Comment:
Ah... I hit `/studies/1/` before I had my URL in place and got a valid response with study stuff in it but that's because the regex isn't terminated by a `$`, so I had some confusion at the time b/c of that. I can adjust that here.